npm/windows.Dockerfile

```diff
@@ -7,7 +7,13 @@ WORKDIR /usr/local/src
 COPY . .
 RUN GOOS=windows CGO_ENABLED=0 go build -v -o /usr/local/bin/azure-npm.exe -ldflags "-X main.version="$VERSION" -X "$NPM_AI_PATH"="$NPM_AI_ID"" -gcflags="-dwarflocationlists=true" npm/cmd/*.go
 
-FROM mcr.microsoft.com/windows/servercore:${OS_VERSION}
+# intermediate for win-ltsc2019
+FROM mcr.microsoft.com/windows/servercore@sha256:6fdf140282a2f809dae9b13fe441635867f0a27c33a438771673b8da8f3348a4 as ltsc2019
```

Comment:
NPM does not run on ltsc 2019. Can update this to just use 2022

Reply:
it will only build the image for 2019 if that's the only OS_VERSION the NPM image build target has